api/invitations.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+categories: ["API"]
+title: Invitations
+---
+
+# Invitations
+
+## Check validity
+
+```sh
+$ curl {{ site.parku.api }}/invitations/ckdzUEdUcWs= \
```

Comment:
yes, this should be non-working, because otherwise users could use that to get a payback ;)